### DISCUSSION TITLE
Использование помощи: функциональность клавиши F7

### DISCUSSION THREAD

**Discussion by Mischutka** [elfmz/far2l#1162] [Q&A]
Нашёл [здесь](https://documentation.help/Far-Manager-ru/documentation.pdf) информацию, как пользоваться помощью FAR Manager. Возникло затруднение при проверке работоспособности клавиши F7 в far2l. По описанию:
> F7 — поиск подстроки в текущем файле помощи. Результаты поиска будут выведены в виде ссылок на темы.
>
> По умолчанию помощь показывается в свёрнутом виде.

С помощью F1 открыл главную страницу файла помощи, перешёл по ссылке в соответствующий раздел, там нажал F7, но ничего не произошло, не обнаружил и свёрнутого окна (вполне возможно, что не там его искал).

Кто-нибудь подскажет, как это должно работать?

**Comment by elfmz** [comment]
far2l был ответвлен от версии far manager'а в которой не было еще этой функциональности, наверное имеет смысл это тоже добавить

**Comment by elfmz** [comment]
функциональность бэкпортирована в текущий мастер в этим коммитом - https://github.com/elfmz/far2l/commit/db8e1fe3581314575b2e1c7f76ca768a6a5e0302

**Comment by Mischutka** [comment]
Также обнаружил ещё две комбинации клавиш, которые не поддерживаются в far2l:

- Ctrl+F1    убрать/показать левую панель
- Ctrl+F2    убрать/показать правую панель

**Comment by elfmz** [comment]
Ctrl+F1/F2 поддерживаются, скорее всего они у вас заняты какимто общесистемными хоткеями.

**Comment by Mischutka** [comment]
Если бы были заняты, то, по идее, при нажатии данных комбинаций должно происходить какое-то действие (как, например, при Alt F4 происходит закрытие окна с far2l вместо вызова стороннего редактора), а так вообще ничего не происходит.

Прошу прощения, данные комбинации действительно заняты:

fly-wmfunc FLYWM_GOTO_PAGING1 и fly-wmfunc FLYWM_GOTO_PAGING2 соответственно.
